tests/loggers/test_wandb.py

```diff
@@ -300,9 +301,9 @@ def wrapper_something():
     params.wrapper_something_wo_name = lambda: lambda: "1"
     params.wrapper_something = wrapper_something
 
-    params = WandbLogger._convert_params(params)
-    params = WandbLogger._flatten_dict(params)
-    params = WandbLogger._sanitize_callable_params(params)
+    params = _convert_params(params)
+    params = _flatten_dict(params)
+    params = _sanitize_callable_params(params)
```

Comment:
Since this test isn't related to wandb at all, let's move it to `tests/utilities/test_logger.py` and add other unit tests as @\ananthsub suggests in https://github.com/PyTorchLightning/pytorch-lightning/pull/11484/files#r789031440?

Reply:
Done!